docs/New-MarkdownHelp.md

```diff
@@ -93,16 +92,15 @@ Mode                LastWriteTime         Length Name
 -a----        5/22/2016   6:56 PM          15320 Set-PSReadlineOption.md
 ```
 
-Create markdown help for inbox PSReadLine module. 
-You don't need to load the module itself to do it.
-Only the help file would be used.
-### Example 4 (from maml file with module page)
+This command creates help in markdown format for the specified help MAML file. You do not have to load the module, as in the previous example. If the module is already loaded, this command creates help based on the MAML file, not on the currently installed module.
+
+
+### Example 4: Create help from an existing MAML file for use in a CAB file
 ```
-PS C:\> $mamlPath = 'C:\Program Files\WindowsPowerShell\Modules\PSReadline\1.1\en-US\Microsoft.PowerShell.PSReadline.dll-help.xml'
-PS C:\> New-MarkdownHelp -OutputFolder .\psreadline-docs -MamlFile $mamlPath -WithModulePage  -Force -ModuleName PSReadLine
+PS C:\> New-MarkdownHelp -OutputFolder "D:\PSReadline\docs" -MamlFile 'C:\Program Files\WindowsPowerShell\Modules\PSReadline\1.1\en-US\Microsoft.PowerShell.PSReadline.dll-help.xml' -WithModulePage  -Force -ModuleName "PSReadLine"
```

Comment:
I broke it into two lines, because one line was too long and it was really hard to read in the rendered from markdown html.